hello-1/.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
``conan test_package`` generates some temp files into test_package/build folder, this avoids to commit them